src/Connection.h

```diff
@@ -32,6 +33,20 @@ class Message;
 }
 }
 
+enum VoiceProtocolType {UNDEFINED, CUSTOM, UDP_OCB2};
+struct VoiceProtocol {
```

Comment:
Why place this struct here? Because I can find a better place :(

Reply:
Why aren't you moving this into its own file `VoiceProtocolType`?

Reply:
Hmmm... Sounds like a good idea. I can move them into VoiceProtocol.h.